src/reaction.ts

```diff
@@ -0,0 +1,306 @@
+import StreamClient, { APIResponse } from './client';
+import StreamFeed from './feed';
+import * as errors from './errors';
+
+export type TargetFeeds = (string | StreamFeed)[];
+
+export type TargetFeed = string | StreamFeed;
+
+export type TargetFeedsExtraData = Record<string, unknown>;
+
+type ReactionBody<T> = {
```

Comment:
Should this have a default instead of data being | Record?

Reply:
with the default value, ts complains about the empty fallback object we have in there